orderedmap.go

```diff
@@ -55,6 +55,22 @@ func (m *OrderedMap) GetOrDefault(key, defaultValue interface{}) interface{} {
 	return defaultValue
 }
 
+// GetElement returns the element for a key. If the key does not exist, the
+// second return parameter will be false and the pointer will be nil.
+func (m *OrderedMap) GetElement(key interface{}) (*Element, bool) {
```

Comment:
I don't think there needs to be a second return value here. The second return value is used in cases where a `nil` would be ambiguous (ie. a `nil` could be mean the the value doesn't exist *or* it does exist but is actually `nil`). In this case the element can only be `nil` if the element doesn't exist.

it also makes it easier and more consistent with the other iterators:

```go
// This is safe to use even if "foo" does not exist.
for e := myMap.GetElement("foo"); e != nil; e = e.Next() {}
```

Reply:
Done! Thanks @elliotchance.